0004-bankless-leader.md

```diff
@@ -103,7 +103,7 @@ empty cache.
 ### Impact on Clients
 
 The same fee account can be reused many times in the same block until it is
-called by system program, or it is passed to a instruction alone side system
+called by system program, or it is passed to a instruction aloneside system
```

Comment:
Wait lol. Sorry to nit pick again, but the comment wasn't about the spacing. It was the spelling. I assume you meant to say "alongside". i.e. "along" vs. "alone"

Reply:
haha, my bad. fixed spelling